### PR TITLE
fix a lot of pep8 whitespace violations

### PR DESCRIPTION
**Pull request by nickpell**
Found these when playing with Pycharm's tools for detecting rule violations.

### CODE REVIEW

File: custom/m4change/signals.py [hunk 1]
```diff
@@ -103,11 +103,11 @@ def _filter_forms(xform, cases):
...
 def handle_m4change_forms(sender, xform, cases, **kwargs):
-     if hasattr(xform, "domain") and xform.domain in M4CHANGE_DOMAINS and hasattr(xform, "xmlns"):
-         if xform.xmlns in ALL_M4CHANGE_FORMS:
+    if hasattr(xform, "domain") and xform.domain in M4CHANGE_DOMAINS and hasattr(xform, "xmlns"):
+        if xform.xmlns in ALL_M4CHANGE_FORMS:
             _handle_duplicate_form(xform, cases)
-         if xform.xmlns in BOOKING_FORMS + BOOKED_DELIVERY_FORMS + UNBOOKED_DELIVERY_FORMS +\
-                 IMMUNIZATION_FORMS + FOLLOW_UP_FORMS:
+        if xform.xmlns in BOOKING_FORMS + BOOKED_DELIVERY_FORMS + UNBOOKED_DELIVERY_FORMS +\
+            IMMUNIZATION_FORMS + FOLLOW_UP_FORMS:
```
[the remark this answers is not in the document]
Comment (esoergel) [reply custom/m4change/signals.py]: this is a legit critique

**Review by esoergel — APPROVED** [review]
:heart:

File: custom/m4change/signals.py (hunk 1 above)
Comment (emord) [inline custom/m4change/signals.py]:
```suggestion
                IMMUNIZATION_FORMS + FOLLOW_UP_FORMS:
```

Comment (emord) [reply custom/m4change/signals.py]: crap. i was hoping i could accept my own suggested chagne...

**Review by emord — APPROVED** [review]

**Review by kaapstorm — APPROVED** [review]

**Review by millerdev — APPROVED** [review]

**Pull request merged**